frontend/app/src/components/settings/controls/SettingsOption.vue

```diff
@@ -83,7 +83,7 @@ const update = async (newValue: any) => {
     setError(result.error);
   }
   emit('finished');
-};
+}, 1500);
```

Comment:
I'm not sure if we should place the debounce here. 
Additionally, what if the input is blurred? Wouldn't it be better to save without any delay?

Reply:
for this, we might reduce the timeout, but it won't be too noticeable on blur. also we don't use the blur event to save for all types

Reply:
Yes update is always attached to the `input` event, so debouncing the update makes more sense because you don't want a network request on every keystroke.